packages/components/Checkbox/src/CheckboxTokens.android.ts

```diff
@@ -0,0 +1,45 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { CheckboxTokens } from './Checkbox.types';
+
+export const defaultCheckboxTokens: TokenSettings<CheckboxTokens, Theme> = (t: Theme) =>
+  ({
+    borderRadius: globalTokens.corner.radius.small,
+    checkboxBorderWidth: 1, // Will be update once new globalTokens for stroke is merged.
```

Comment:
It would be best to wait fo Lynn's  PR to go in before closing this one

Reply:
Thanks Ankita - would be great if this PR waited for mine but unfortunately there are some blockers on my PR and I'm not sure how long those will take to get resolved. If this needs to go in first I think that should be fine, I can do the merge.

Reply:
The tokens that would get impacted in this PR: corner radius and font size tokens would be renamed

Reply:
Thanks Lynn, yeah I think we can checkin this PR in once Rohan takes a look back on open comments if our PR takes longer.

Reply:
@ankraj12 @lyzhan7  , I will check in this initial set of tokens and work upon functionality check till then, will update when @lyzhan7 PR goes in.